alsa.te

```diff
@@ -30,15 +36,13 @@ files_type(alsa_var_lib_t)
 type alsa_var_lock_t;
 files_lock_file(alsa_var_lock_t)
 
-type alsa_home_t;
-userdom_user_home_content(alsa_home_t)
-
 ########################################
 #
 # Local policy
 #
 
-allow alsa_t self:capability { dac_read_search dac_override setgid setuid ipc_owner };
+# kill : kill itself
+allow alsa_t self:capability { dac_read_search dac_override kill setgid setuid ipc_owner };
```

Comment:
The `kill` capability is for killing a process with another UID.  That doesn't seem like it should apply here.